### PR TITLE
Show loading/complete state on each step in `StepBubble`

### PR DESCRIPTION
**Pull request by cryptickoan**
Show loading/complete state on each step in `StepBubble`

### CODE REVIEW

File: components/Progress/Progress.tsx [hunk 1]
```diff
@@ -48,7 +54,7 @@ const Progress: React.FC<ProgressProps> = ({
   return (
     <Box position="relative">
       <ChakraProgress
-        height={8}
+        height={barHeight ?? 8}
```
Comment (nathanhleung) [inline components/Progress/Progress.tsx:57]: if you take a look at line 64 (`...restProps`), you're already able to override the `height` prop by passing it to the component itself

e.g. `<Progress height={10} />` in the dapp will pass `10` through automatically here without the need for an extra prop for `barHeight` to be manually added

lmk if that doesn't work though and i can investigate further

File: components/StepBubbles/StepBubbles.tsx [hunk 2]
```diff
@@ -10,9 +10,13 @@ type StepBubbleProps = {
    * Whether this bubble is the last in line and should not have a connector.
    */
   last?: boolean;
+  /**
+   * Whether the bubble should show a spinner if its active.
+   */
+  spinner?: boolean;
```
Comment (nathanhleung) [inline components/StepBubbles/StepBubbles.tsx:16]: Is the use case for the spinner to denote when a step is in progress? In that case, I might rename this prop `loading` and also add it up in the `StepBubbles` component too, then pass it thru to the `StepBubble`

e.g. `<StepBubbles steps={3} activeIndex={1} loading />` would show a spinner on the second step, with no need for a dev to manually set the `spinner` prop in `StepBubble` itself when consuming the exported `StepBubbles` component.

My rationale is that `StepBubble` isn't exported by this file, so I would think of it more like a private component whose implementation details are hidden. I would rename the prop to `loading` to more clearly indicate the intended use case and to abstract the idea of a loading state from its implementation (showing a `<Spinner />`).

Let me know your thoughts/if you have any questions!

**Review by nathanhleung — COMMENTED** [review]
thanks for the pr! left a few comments

File: components/Progress/Progress.tsx (hunk 1 above)
Comment (cryptickoan) [reply components/Progress/Progress.tsx]: Oh yikes yeah just reverted the changes, thanks!

File: components/StepBubbles/StepBubbles.tsx (hunk 2 above)
Comment (cryptickoan) [reply components/StepBubbles/StepBubbles.tsx]: This makes a lot of sense yes. I updated the component to use loading from StepBubbles

**Pull request merged**